comp/src/main/java/frc/robot/subsystems/swervedrive/SwerveCommands.java

```diff
@@ -199,6 +217,34 @@ public Command autoAlignAndIntakeAlgae(double setpoint) {
         return command;
     }
 
+    public Pose2d getClimbPose2d() {
```

Comment:
@anherm160 figured out the issue you running into last meeting. Switch statements are weird, and unlike if statements each case is checked. That means unless there's a `return` or `break` in the cases, the code in default will always be run which in this case meant the robot would always align to the pose in the default case. I updated the function to return the pose instead of assigning it to a variable so this doesn't happen.